public/js/main.js

```diff
@@ -79,17 +63,17 @@ function getSelectedTab(tabs) {
   return tabs.find(tab => tab.get("id").includes(childId));
 }
 
-setTimeout(function() {
-  if (!getTabs(store.getState()).isEmpty()) {
-    return;
-  }
+// setTimeout(function() {
+//   if (!getTabs(store.getState()).isEmpty()) {
+//     return;
+//   }
 
-  ReactDOM.render(
-    React.DOM.div({ className: "not-connected-message" },
-      "Not connected to Firefox"
-    ),
-    document.querySelector("#mount"));
-}, 500);
+//   ReactDOM.render(
+//     React.DOM.div({ className: "not-connected-message" },
+//       "Not connected to Firefox"
+//     ),
+//     document.querySelector("#mount"));
+// }, 500);
```

Comment:
I was running into some weird problems and commented this out to make sure it wasn't that problem, but it wasn't related. I don't see any problem leaving this in here unconditionally, but we could put it behind a flag as well.